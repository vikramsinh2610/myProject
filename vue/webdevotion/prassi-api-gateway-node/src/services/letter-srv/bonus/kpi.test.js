const KPI = require('./kpi');

describe('KPI', () => {
  test('should have these properties', () => {
    const kpi = new KPI({ _id: 'de2f234-2d2ed2-3d2232' });
    expect(kpi).toHaveProperty('_id');
    expect(kpi).toHaveProperty('value');
  });

  test('should return it s JSON Schema', () => {
    expect(KPI.getJSONSchema()).toMatchSnapshot();
  });
});
