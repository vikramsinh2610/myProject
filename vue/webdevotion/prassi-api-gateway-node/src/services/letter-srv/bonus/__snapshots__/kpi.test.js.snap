// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KPI should return it s JSON Schema 1`] = `
Object {
  "properties": Object {
    "_id": Object {
      "type": "string",
    },
    "options": Object {
      "additionalProperties": true,
      "type": "object",
    },
    "type": Object {
      "enum": Array [
        "number",
        "currency",
        "percentage",
      ],
      "type": "string",
    },
    "value": Object {
      "type": "integer",
    },
  },
  "required": Array [
    "_id",
  ],
  "type": "object",
}
`;
